addons/mass_mailing/models/mass_mailing.py

```diff
@@ -315,6 +316,18 @@ def _get_source_id(self, cr, uid, context=None):
     def _get_medium_id(self, cr, uid, context=None):
         return self.pool['ir.model.data'].xmlid_to_res_id(cr, uid, 'utm.utm_medium_email')
 
+    def create(self, cr, uid, vals, context=None):
+        mass_mailing_campaign_id = super(MassMailingCampaign, self).create(cr, uid, vals, context=context)
+        mass_mailing_campaign = self.pool['mail.mass_mailing.campaign'].browse(cr, uid, mass_mailing_campaign_id, context=context)
+        self.pool['utm.campaign'].write(cr, uid, [mass_mailing_campaign.campaign_id.id], {'name': vals['name']}, context=context)
```

Comment:
Does that work ? Because name is required on utm.campaign ... so it seems impossible to create a campaign without name.